zellij-server/src/tab/mod.rs

```diff
@@ -152,6 +152,7 @@ pub trait Pane {
     fn push_right(&mut self, count: usize);
     fn pull_left(&mut self, count: usize);
     fn pull_up(&mut self, count: usize);
+    fn dump_screen(&mut self, client_id: ClientId) -> String;
```

Comment:
Instead of the empty implementation in `PluginPane` we can add a default implementation here.
Something like:

```rust
fn dump_screen(&mut self, client_id: ClientId) -> String {
    return "".to_owned();
}
```
I think it'll be a little cleaner